eth/filters/filter.go

```diff
@@ -32,13 +32,10 @@ type AccountChange struct {
 
 // Filtering interface
 type Filter struct {
-	db       ethdb.Database
-	earliest int64
-	latest   int64
-	skip     int
-	address  []common.Address
-	max      int
-	topics   [][]common.Hash
+	db         ethdb.Database
+	begin, end int64
+	address    []common.Address
```

Comment:
I think it can be improved. It's called `address` but in fact refers to multiple addresses, since it's slice of addresses.

It looks a bit strange when used in a range expression, for example:
```
for _, addr := range self.address {
```
would turn into:
```
for _, addr := range self.addresses {
```